scripts/gcc/4.9.2-cortex_a9-hf/script.sh

```diff
@@ -28,4 +28,8 @@ function mason_ldflags {
     :
 }
 
+function mason_static_libs {
```

Comment:
Is this because when you want to link statically with the tool chain you do it via `-static` ?